### DISCUSSION TITLE
Yahoo finance API not connecting

### DISCUSSION THREAD

**Discussion by xperus** [tarkah/tickrs#149] [Q&A]
It appears that there no connectivity to the API (Yahoo possibly changed something). No stock data is populating for any of  ticker targets. 
![Screenshot at 2023-09-01 08-50-43](https://github.com/tarkah/tickrs/assets/67400911/6db11486-645a-4d4f-acf3-5f3d8b0dee7e)

**Comment by tarkah** [comment]
Are you on the latest version? https://github.com/tarkah/tickrs/releases/tag/v0.14.9

**Reply by xperus** [reply]
The new version has cleared up all issues, Its working perfectly. Thanks

**Comment by NCPROBST** [comment]
I just downloaded 0.14.9 for aarch64 and it does not update when I add a stock. If I however press 'o' it shows the correct value but only in the options pane. In the main pane it is infinitely loading ..... So there is definitely some sort of connection to the api.

**Reply by tarkah** [reply]
Do you have an adblocker / pihole running? If so, it might be blocking a domain that this tries to get a cookie from, fc.yahoo.com

**Reply by NCPROBST** [reply]
Yes I do have a pihole running. Whitelisting fc.yahoo.com fixed the issue. Thank you! :)
